features/fields-filtered-Persoon.csv

```diff
@@ -0,0 +1,128 @@
+pad
+aNummer
+adressering
+adressering.adresregel1
+adressering.adresregel2
+adressering.adresregel3
+adressering.land
+adressering.aanhef
+adressering.aanschrijfwijze
+adressering.aanschrijfwijze.aanspreekvorm
+adressering.aanschrijfwijze.naam
+adressering.gebruikInLopendeTekst
+burgerservicenummer
+datumEersteInschrijvingGBA
+datumInschrijvingInGemeente
+europeesKiesrecht
+europeesKiesrecht.aanduiding
+europeesKiesrecht.einddatumUitsluiting
+geboorte
+geboorte.datum
+geboorte.land
+geboorte.plaats
+gemeenteVanInschrijving
+geslacht
+immigratie
+immigratie.datumVestigingInNederland
+immigratie.indicatieVestigingVanuitBuitenland
+immigratie.landVanwaarIngeschreven
+immigratie.vanuitVerblijfplaatsOnbekend
+indicatieCurateleRegister
+indicatieGezagMinderjarige
+kinderen
+kinderen.burgerservicenummer
+kinderen.geboorte
+kinderen.geboorte.datum
+kinderen.geboorte.land
+kinderen.geboorte.plaats
+kinderen.naam
+kinderen.naam.adellijkeTitelPredicaat
+kinderen.naam.adellijkeTitelPredicaat.soort
```

Comment:
volgens mij moet het soort veld ook hier weg

Reply:
goed gezien! inmiddels verwijderd (door AdellijkeTitelPredicaatType toe te voegen in de configuratie van "autoComponents")